utils/cmpcodesize/cmpcodesize/__init__.py

```diff
@@ -1,6 +1,3 @@
-from __future__ import absolute_import
-from .main import main
-
```

Comment:
I'm curious, though: is it bad form to use this style of relative imports? If so, should a corresponding change be proposed to llvm/utils/lit as well?

/cc @ddunbar, author of lit